trie-db/Cargo.toml

```diff
@@ -8,15 +8,15 @@ license = "Apache-2.0"
 
 [dependencies]
 log = "0.4"
-rand = { version = "0.7", default-features = false }
+rand = { version = "0.7", default-features = false, features = [ "small_rng" ] }
```

Comment:
now that I think about it, should it be only a dev dependency?

Reply:
Doesn't [look](https://github.com/paritytech/trie/blob/master/trie-db/src/lib.rs#L23) so.

Reply:
But is it actually **used**?

Reply:
right, I think it's best to remove this change from this PR at all and move it to dev dependencies in #67